code/EDA.Rmd

```diff
@@ -110,90 +88,52 @@ Filtering in R is like adding conditions with WHERE in SQL.
 Plotting is rather simple and intuitive. 
 
 Let's visualize the city and highway mileage (mile per gallon of fuel).
-The ggplot library acts in an additive way. You can add more and amore to the figure. 
-Task: plot city vs highway mileage. Show engine size and vehicle class as size and color. 
```

Comment:
I thought we spoke about using the same dataset? This probably makes it more difficult for us and the participants to check the correctness of all tasks and code. We should probably stick to just 1 dataset (the one in folder `data/` and do all analysis tasks on that.